t/lib/t/MusicBrainz/Server/Controller/UnconfirmedEmailAddresses.pm

```diff
@@ -84,6 +84,8 @@ test 'Paths that allow browsing without a confirmed email address' => sub {
   "Controller::Admin::delete_user",
   "Controller::Admin::edit_banner",
   "Controller::Admin::edit_user",
+  "Controller::Admin::email_search",
+  "Controller::Admin::ip_lookup",
```

Comment:
I'm not sure why these *can* be seen with an unconfirmed email address, but I guess that's true of all admin bits anyway, so it's not inconsistent.

Reply:
Yeah. I also thought this was silly, but it's technically true that somebody can have an admin flag set without a confirmed email address, so...